components/AcceptPayments/PaymentAPI.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
```

Comment:
The  content here too should be written as MDX @asapconet  can you work with @gideonnnalue  to update the codeblock to the new one and also write this content as MDX?

Reply:
No issues, will do just that.